### PR TITLE
doc/md: documenting creation of encryption keys

### PR DESCRIPTION
**Pull request by rotemtam**

### CODE REVIEW

**Review by zeevmoney — APPROVED** [review]

File: doc/md/deployment/intro.md [hunk 1]
```diff
@@ -58,4 +58,29 @@ Let's review this command step by step:
 4. ```5800:5800 arigaio/atlas:latest``` use the latest atlas image, for a versioned release use a specific tag such as ```arigaio/atlas:0.3.6```.
 5. ```serve --addr :5800 --storage "mysql://root:pass@tcp(host.docker.internal:3306)/atlas"``` serve Atlas on port 5800 with a persistent MySQL Database.  
 
+### Encryption
 
+Atlas uses [tink](https://github.com/google/tink), a battle-tested encryption library created at Google, to 
+encrypt all sensitive information.  Following recommendations from the developers of tink, Atlas
```
Comment (a8m) [inline doc/md/deployment/intro.md:64]:
```suggestion
encrypt all sensitive information. Following recommendations from the developers of tink, Atlas
```

File: doc/md/deployment/intro.md [hunk 2]
```diff
@@ -58,4 +58,29 @@ Let's review this command step by step:
...
+uses [AEAD](https://developers.google.com/tink/aead?hl=en) encryption with an AES256_GCM type key.
+
+On its first run, Atlas will generate a keyset.json file under `$HOME/.atlas/`  containing
```
Comment (zeevmoney) [inline doc/md/deployment/intro.md:67]:
```suggestion
On its first run, Atlas will generate a keyset.json file under `$HOME/.atlas/` containing 
```

Comment (a8m) [inline doc/md/deployment/intro.md:67]:
```suggestion
On its first run, Atlas generates a keyset.json file under `$HOME/.atlas/`  containing 
```

**Review by a8m — APPROVED** [review]
💯

**Pull request merged**